docs/concepts/overview/working-with-objects/common-labels-annotations.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Recommended Labels and Annotations
+---
+
+{% capture overview %}
+Many tools can be used to visualize and manage Kubernetes objects. The tools used
```

Comment:
How does this relate to the Application KEP from @kow3ns?  There seems to be some overlap in goals.  Is the vision that the Application resource would enforce these conventions?

Reply:
Is there something more than what's in my comment on the KEP at https://github.com/kubernetes/website/pull/7695#issuecomment-378621396 you'd like to dig into?